andela/getmyshop
# 107803086 Shop owners authentication

#### What does this PR do?

It authenticates shop owners.
#### Description of Task to be completed?

Enables shop owners sign in and then redirects them to their dashboard.
#### How should this be manually tested?

Visit http://getmyshop-staging-pr-106.herokuapp.com/shop_login and fill the sign in form with your registered account. Upon successful sign in, you should be redirected to http://getmyshop-staging-pr-106.herokuapp.com/dashboard.
#### What are the relevant pivotal tracker stories?
#107803086
